docs/_docs/maintaining/releasing-a-new-version.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Releasing a new version"
+---
+
+**This guide is for maintainers.** These special people have **write access** to one or more of Jekyll's repositories and help merge the contributions of others. You may find what is written here interesting, but it’s definitely not for everyone.
+{: .note .info }
+
+The most important thing to understand before making a release is that there's no need to feel nervous. Most things are revertable, and even if you do publish an incomplete gem version, we can always skip that one. Don't hestitate to contact the other maintainers if you feel unsure or don't know what to do next.
+
+### Bump the version
+
```

Comment:
A shorter overview of the release process 

1. Update manually `lib/jekyll/version.rb` and bump version according to SemVer (e.g, `3..8.0`)
2. Update manually `History.markdown` and replace current HEAD
2. Run `bundle exec rake site:releases:new[3.8.0]` and write release post (submit the post to other maintainers review). Thanks and mention all collaborators since latest version with `git shortlog -sn master...v3.7.2`
4. Run `bundle exec rake site:generate` to update `docs/latest_version.txt`, `docs/_config.yml` and generate History page.
5. `git add . && git commit -m  "Release :gem: v3.8.0"`.
6. `bundle exec rake release`
7.  Tweet and celebrate 🎉

Reply:
@DirtyF Done!

Reply:
What about the 4th point?

Reply:
oh whoops, forgot about changing the command there